api/v1alpha1/jobset_webhook_test.go

```diff
@@ -21,7 +33,11 @@ func TestJobSetDefaulting(t *testing.T) {
 					ReplicatedJobs: []ReplicatedJob{
 						{
 							Template: batchv1.JobTemplateSpec{
```

Comment:
Could you create this template once and reuse everywhere?

Reply:
makes sense, done